.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "ruby.rubocop.executePath": "/usr/local/bin/rubocop"
+}
```

Comment:
I don't think this file should have been committed. It may be wise to include your IDE's files in your personal `.gitignore` to avoid this in the future.